packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -22,6 +22,21 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
 
   private _classNames: IClassNames<IGroupHeaderStyles>;
 
+  public static getDerivedStateFromProps(newProps: IGroupHeaderProps, state: IGroupHeaderState): IGroupHeaderState {
```

Comment:
Any reason for not calling these `nextProps` and `prevState`?

Reply:
Nope, good suggestion! Bit clearer for the reader.